src/models.py

```diff
@@ -40,6 +40,12 @@ class StatusEnum(str, Enum):
 	audit_failed = 'audit_failed'
 
 
+class DataAccessEnum(str, Enum):
+	public = 'public'
+	private = 'private'
+	viewonly = 'viewonly'
```

Comment:
Maybe we consider using a more declarative name here as 'Data' can mean too many things in the application.  We have the original images (lacking a really good name here, we call them sometimes ortho, sometimes 'archive', sometimes 'images), the cogs, and then there is the Dataset, which consists of ortho, labels and metadata. 
Does this data access apply to orthos only, or also to the cogs. Then I suggest calling it 'DatasetAccessEnum'. I don't really care for the name, I just think 'Data' is too generic.